oap-server/server-receiver-plugin/receiver-proto/src/main/fbs/istio/node-info.fbs

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2020 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
You copied this file from Istio, right? You should update the LICENSE of source codes, like the proto files we copied.

Reply:
Can you please kindly recheck?

Reply:
The license is clear. Just recommend enhancing the documentation about open ALS and metadata exchange activation.
Even in the demo, it is activated in default, we need to highlight that, and reference to the e2e.